src/sounddeviceportaudio.cpp

```diff
@@ -765,6 +750,25 @@ int SoundDevicePortAudio::callbackProcessClkRef(const unsigned int framesPerBuff
     if (!m_bSetThreadPriority) {
         QThread::currentThread()->setPriority(QThread::TimeCriticalPriority);
         m_bSetThreadPriority = true;
+
+        // This disables the denormals calculations, to avoid a
+        // performance penalty of ~20
+        // https://bugs.launchpad.net/mixxx/+bug/1404401
+#ifdef __SSE__
```

Comment:
Yes SSE and MMX is a subset of SSE2 and SSE2 is not strictly required to have these flags.